builtin/game/falling.lua

```diff
@@ -52,6 +52,31 @@ core.register_entity(":__builtin:falling_node", {
 		local pos = self.object:getpos()
 		-- Position of bottom center point
 		local bcp = {x = pos.x, y = pos.y - 0.7, z = pos.z}
+		-- Check for player/mobs below falling node and hurt them >:D
+		if self.timer and self.timer > 0 then
+			self.timer = self.timer - dtime
+		else
+			-- Get damage level from falling_node group (when above 1)
+			local damage = core.registered_nodes[self.node.name] and
+				core.registered_nodes[self.node.name].groups.falling_node
+			if damage and damage > 1 then
+				local all_objects = minetest.get_objects_inside_radius(bcp, 0.7)
+				for _,obj in ipairs(all_objects) do
```

Comment:
Speed depends on the implementation. While speed here is very important I don't see a way to make it both readable and fast in all implementations.

This is fast enough.